src/acceptanceTest/java/uk/gov/hmcts/futurehearings/hmi/acceptance/directlisting/POSTDirectListingValidationTest.java

```diff
@@ -0,0 +1,36 @@
+package uk.gov.hmcts.futurehearings.hmi.acceptance.directlisting;
+
+import lombok.extern.slf4j.Slf4j;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.TestInstance;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.HttpStatus;
+import org.springframework.test.context.ActiveProfiles;
+import uk.gov.hmcts.futurehearings.hmi.Application;
+import uk.gov.hmcts.futurehearings.hmi.acceptance.common.verify.error.HMICommonErrorVerifier;
+import uk.gov.hmcts.futurehearings.hmi.acceptance.common.verify.success.HMICommonSuccessVerifier;
+
+@Slf4j
```

Comment:
Add this @TestInstance(TestInstance.Lifecycle.PER_CLASS) to class to fix the error "'initialiseValues()' has incorrect signature for a @org.junit.jupiter.api.BeforeAll method "

Reply:
@ShufenZ please see line 18

Reply:
Sorry, my comment should be for the file above this one.